django/contrib/admin/static/admin/js/cancel.js

```diff
@@ -14,10 +14,10 @@
     ready(function() {
         function handleClick(event) {
             event.preventDefault();
-            if (window.location.search.indexOf('&_popup=1') === -1) {
-                window.history.back(); // Go back if not a popup.
+            if (window.location.search.includes('&_popup=1')) {
```

Comment:
Unrelated to the change, but is there a bug here? What about `?_popup=1`?

Reply:
Looks that way to me, yeah. Good eye.

This and other querysetring code should probably use [URLSearchParams.has()](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/has) rather than trying to parse the querystring manually.

Reply:
Fixed in #13101.

During my testing, I was unable to trigger this bug in practice. It turns out, the URL always has the `&` in it. Either way, we should get ahead of this before it does accidentally become a problem. Thanks again for catching.